designer/server/src/models/forms/submission-guidance.js

```diff
@@ -0,0 +1,36 @@
+import { buildErrorList } from '~/src/common/helpers/build-error-details.js'
+
+/**
+ * @param {FormMetadata} metadata
+ * @param {ValidationFailure<Pick<FormMetadataInput, 'submissionGuidance'>>} [validation]
+ */
+export function submissionGuidanceViewModel(metadata, validation) {
+  const pageTitle = 'Tell users what happens after they submit their form'
+  const { formValues, formErrors } = validation ?? {}
+
+  return {
+    form: metadata,
+    backLink: {
+      text: 'Back',
+      href: `/library/${metadata.slug}`
+    },
```

Comment:
I think we're suppose to copy what the editor page does?

Quite a few back links that don't though

```suggestion
    backLink: {
      text: 'Back to form overview',
      href: `/library/${metadata.slug}`
    },
```